share/src/main/webapp/components/preview/PdfJs.js

```diff
@@ -1902,7 +1913,7 @@
       this.textLayerDiv = null;
       this.config = config || {};
       this.textContent = null;
-      this.textLayerDiv = null;
+      this.annotationLayerDiv = null;
```

Comment:
It seems you removed the parameter variable **textLayerDiv** here

Reply:
Forget it. You created a copy of the original one, and then changed it to annotationLayerDiv... sorry

Reply:
@douglascrp well... I reused an existing (duplicate property), instead of adding a new one..